org.jacoco.core/src/org/jacoco/core/analysis/Analyzer.java

```diff
@@ -108,6 +108,7 @@ public void visitEnd() {
 	public void analyzeClass(final ClassReader reader) {
 		final ClassVisitor visitor = createAnalyzingVisitor(
 				CRC64.checksum(reader.b), reader.getClassName());
+		// TODO maybe we can SKIP_FRAMES here?
```

Comment:
@marchof I removed comment from this PR as anyway it will be presented in #500 , so that we can come back to this later.